### PR TITLE
User Profile Pages, Global Responsive Nav, Responsive User Profile Nav

### PR DESCRIPTION
**Pull request by Razorback360**

### CODE REVIEW

**Comment by Razorback360** [comment]
image
image
image
image
image
image

File: client/src/pages/profile/donations.tsx
Comment (Yokozuna59) [inline client/src/pages/profile/donations.tsx:1]: React JSX file names should start with caps letters. Rename the file to `Donations.tsx`. Same goes to `Donation.css`.

File: client/src/pages/profile/mobile.tsx
Comment (Yokozuna59) [inline client/src/pages/profile/mobile.tsx:1]: React JSX file names should start with caps letters. Rename the file to `Mobile.tsx`.

**Review by Yokozuna59 — CHANGES_REQUESTED** [review]
Please delete `npm` related files (e.g. `package-lock.json`) and use `pnpm` instead.
Also, please format files and remove unused imports.

File: client/src/pages/profile/donations.tsx
Comment (Razorback360) [reply client/src/pages/profile/donations.tsx:1]: resolved

File: client/src/pages/profile/mobile.tsx
Comment (Yokozuna59) [reply client/src/pages/profile/mobile.tsx:1]: If you say so. The syntax highlighting for those file isn't working so I assumed it wasn't on purpose.

**Review by Yokozuna59 — COMMENTED** [review]
There's still some `npm` related stuff:

- https://github.com/Razorback360/SWE363-Project/blob/5c572a12d6652f104939a19d56582306260b88b6/client/package.json#L15
- https://github.com/Razorback360/SWE363-Project/blob/5c572a12d6652f104939a19d56582306260b88b6/server/package.json#L17

**Review by Yokozuna59 — APPROVED** [review]

**Pull request merged**